docker/xtask.dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rust:1.76-slim-bullseye as builder
+
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    build-essential \
+    protobuf-compiler \
+    libclang-dev \
+    git \
+    pkg-config \
+    libssl-dev
+
+COPY . .
+
+RUN cargo build --release --bin xtask
+
+FROM debian:bullseye-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    build-essential \
+    protobuf-compiler \
+    libclang-dev \
+    git \
+    pkg-config \
+    libssl-dev
+
+COPY --from=builder /usr/src/app/target/release/xtask /app
+RUN chmod +x xtask
+RUN ln -s /app/xtask /usr/local/bin/xtask
```

Comment:
Keep the actual binary on `/usr/local/bin/`. Otherwise a external bind mount on `/app/` could just make the folder empty.

> If you bind mount file or directory into a directory in the container in which files or directories exist, the pre-existing files are obscured by the mount.

https://docs.docker.com/engine/storage/bind-mounts/#bind-mounting-over-existing-data